ansible_ai_connect/ai/api/model_pipelines/langchain/pipelines.py

```diff
@@ -280,7 +320,86 @@ def __init__(self, config: LANGCHAIN_PIPELINE_CONFIGURATION):
         super().__init__(config=config)
 
     def invoke(self, params: RoleGenerationParameters) -> RoleGenerationResponse:
-        return "dummy_role", [], "dummy_outline"
+        request = params.request
+        text = params.text
+        additional_context = params.additional_context
+        create_outline = params.create_outline
+        outline = params.outline
+        model_id = params.model_id
+        generation_id = params.generation_id
+        file_types = params.file_types
+
+        SYSTEM_MESSAGE_TEMPLATE = """
+        You are an ansible expert optimized to generate Ansible roles.
+        The first print the role name in a way: role_name.
```

Comment:
If this works, it works.

It isn't very clear English though. 

"First print the role..."?

"The first part prints the role..."?